### ISSUE TITLE
Guidebook can't render entities that aren't 32x32

### ISSUE THREAD

**Issue by Elijahrane** [space-wizards/space-station-14#13638]
## Description

It also shows some of the wrong states (such as fill layer stuff when it should just pick empty fill appearance)
**Screenshots**

![image](https://user-images.githubusercontent.com/60792108/213883156-b8841b06-52ec-4f0e-ba99-2cc0d85b0deb.png)

**Comment by moonheart08** [comment]
Anything with the entity visuals itself has absolutely nothing to do with guidebooks, it's a symptom of that entity not setting up it's visuals on the client properly.

**Comment by ElectroJr** [comment]
it just uses sprite views to show entities, so the lack of support for large entities is just an engine / sprite view issue. See also space-wizards/RobustToolbox/issues/925 and space-wizards/RobustToolbox/issues/3071.

And or appearance stuff, it just spawns a client-side entity in null-space and shows that. If your entity relies on server-side code to set up some appearance stuff, it won't work. E.g., this is why all the tools in the guidebook are white, because the color randomization is done server-side. Best fix is probably to just give the entity prototype reasonable default values, even if the server usually overrides them.

**Comment by Elijahrane** [comment]
> Anything with the entity visuals itself has absolutely nothing to do with guidebooks, it's a symptom of that entity not setting up it's visuals on the client properly. (In other words, your Oracle entity is broken)

it uses unmodified upstream solution container appearance visuals

**Comment by EmoGarbage404** [comment]
> it uses unmodified upstream solution container appearance visuals
just replace `oracle-1` with `oracle-0` and it should look fine.
``` 
 - type: Sprite
    netsync: false
    noRot: true
    drawdepth: Mobs
    offset: "0.0,0.5"
    sprite: Nyanotrasen/Structures/Decoration/statues.rsi
    layers:
      - state: oracle-0
      - map: ["enum.SolutionContainerLayers.Fill"]
        state: oracle-1
```

**Comment by ElectroJr** [comment]
The sprite views should've been fixed by #15881. Note that it did have a side effect that guide book sprite views now scale with the size of the sprite of the entity, but that could be fixed by just setting the sprite view size to some fixed number.
